subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -51,6 +51,18 @@
 #include "ull_vendor.h"
 #endif /* CONFIG_BT_CTLR_USER_EXT */
 
+/**
+ *  User CPR Interval
+ */
+#if !defined(CONFIG_BT_CTLR_USER_CPR_INTERVAL)
+/* Bluetooth defined CPR Interval Minimum (7.5ms) */
+#define CONN_INTERVAL_MIN(x) (6)
+#else /* CONFIG_BT_CTLR_USER_CPR_INTERVAL */
+/* Proprietary user defined CPR Interval Minimum */
+extern uint16_t ull_conn_interval_min_get(struct ll_conn *conn);
+#define CONN_INTERVAL_MIN(x) (ull_conn_interval_min_get(x))
```

Comment:
```suggestion
#define CONN_INTERVAL_MIN(x) MAX(ull_conn_interval_min_get(x), 6)
```

Reply:
The purpose is to allow a proprietary protocol a connection interval that is outside the valid range so also less than 6. Maybe it wasn't clear but if you enabled the feature the ull_conn_interval_min_get function would have to look at the given connection and return a suitable minimum connection interval, so 6 for a proper Bluetooth LE connection and maybe something less for a proprietary protocol.

Reply:
Fine by me. Could you check if controller can gracefully handle the value 0 as the minimum interval?

Reply:
Clamped it to a minimum of 1.